.github/workflows/ci.yml

```diff
@@ -145,17 +145,50 @@ jobs:
       fail-fast: false
       matrix:
         vtk_version: [commit, v9.3.0, v9.2.6]
-        cpu: [x86_64, arm64]
+        cpu: [x86_64]
         bundle_label: [no-bundle]
         include:
           - vtk_version: commit
             bundle_label: bundle
             cpu: x86_64
+
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        path: 'source'
+        fetch-depth: 0
+        lfs: false
+
+    - name: Generic CI
+      uses: ./source/.github/actions/generic-ci
+      with:
+        vtk_version: ${{matrix.vtk_version}}
+        bundle_label: ${{matrix.bundle_label}}
+        cpu: ${{matrix.cpu}}
+        lfs_sha: ${{ needs.cache_lfs.outputs.lfs_sha}}
+
+#----------------------------------------------------------------------------
+# MacOS ARM CI: Build and test, cross-vtk build matrix
+#----------------------------------------------------------------------------
+  macos_arm:
+    if: github.event.pull_request.draft == false
+    needs: cache_lfs
+
+    strategy:
+      fail-fast: false
+      matrix:
+        vtk_version: [commit]
+        cpu: [arm64]
+        bundle_label: [no-bundle]
```

Comment:
Can you create a bundle/no-bundle matrix instead of the include field here?

Reply:
I think I should still test 9.3.0 and 9.2.6, I forgot to add them back.

I can remove bundle if you want though.